build/Makefile

```
@@ -4,17 +4,18 @@ export PMM_SERVER_IMAGE ?= docker.io/perconalab/pmm-server:3-dev-latest
 
 ## ----------------- PACKER ------------------
 fetch:
-	mkdir -p ${PACKER_CACHE_DIR}/box || :
+	mkdir -p ${PACKER_CACHE_DIR}/box
 	test -f ${PACKER_CACHE_DIR}/id_rsa_vagrant \
 	    || curl -L https://raw.githubusercontent.com/hashicorp/vagrant/master/keys/vagrant \
 		-o ${PACKER_CACHE_DIR}/id_rsa_vagrant
 	chmod 600 ${PACKER_CACHE_DIR}/id_rsa_vagrant
-	test -f ${PACKER_CACHE_DIR}/box/oracle9.ova \
-		|| curl -fL https://pmm-build-cache.s3.us-east-2.amazonaws.com/VBOXES/oracle9-202401.31.0.box -o ${PACKER_CACHE_DIR}/box/oracle9.ova
 
-	# NOTE: image from vagrant registry is twice as large
+	# Add the box using Vagrant
 	test -f ${PACKER_CACHE_DIR}/box/box.ovf \
-		|| tar -C ${PACKER_CACHE_DIR}/box -xvf ${PACKER_CACHE_DIR}/box/oracle9.ova
+		|| VAGRANT_HOME=${PACKER_CACHE_DIR}/box vagrant box add bento/oraclelinux-9 --box-version 202407.23.0 --provider virtualbox
```

```suggestion
		|| VAGRANT_HOME=${PACKER_CACHE_DIR}/box vagrant box add bento/oraclelinux-9 --box-version ${BOX_VERSION} --provider virtualbox
```
Since we have to update the box version from time to time, I suggest we make it a variable.